src/debugpy/server/adapters.py

```diff
@@ -60,6 +62,7 @@ class Expectations(components.Capabilities):
     _capabilities: Capabilities = None
     _expectations: Expectations = None
     _start_request: messaging.Request = None
+    _goto_targets_map: IDMap = IDMap()
```

Comment:
I would recommend using `new_dap_id()` for this. The main reason why I made that a single global counter is because that makes it so much easier to grep things in the message log when IDs are not reused.

Reply:
You mean inside of the IDMap.

Reply:
I used `new_dap_id` for the `next_id` function in the IDMap